packages/react-components/src/components/ChatMessage/ChatMessageComponent.tsx

```diff
@@ -48,12 +53,16 @@ export const ChatMessageComponent = (props: ChatMessageComponentProps): JSX.Elem
 
   const onEditClick = useCallback(() => setIsEditing(true), [setIsEditing]);
 
-  const { onDeleteMessage, message } = props;
+  const { onDeleteMessage, onSendMessage, message } = props;
   const onRemoveClick = useCallback(() => {
     if (onDeleteMessage && message.messageId) {
       onDeleteMessage(message.messageId);
     }
   }, [message.messageId, onDeleteMessage]);
+  const onResendClick = useCallback(() => {
```

Comment:
(Potentially a small thing, not blocking)
i realize I am just looking at this ____Click function now its kind of wierd the on____ functions have Click at the end. Is this right?

Reply:
I think the function name is trying to say on click resend? I was just following the naming patterns of other existing functions (onremoveclick, onupdateclick ...) but yeah now you mentioned it, it does seems a bit weird